environment.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
   - mkdocs
   - mkdocs-material
   - mkdocs-material-extensions
+  - jupyter
```

Comment:
This dependency is not strictly necessary, but it does make it possible to run the notebook in the documentation environment, which is helpful.